tests/test_block_exact_match.py

```diff
@@ -7,25 +7,24 @@
 from hivemind import P2PHandlerError
 from test_utils import *
 
+import src
 from src.bloom.from_pretrained import load_pretrained_block
-from src.client.remote_block import RemoteTransformerBlock
+from src.client.remote_sequential import RemoteTransformerBlock
 from src.data_structures import UID_DELIMITER
 from src.dht_utils import get_remote_module
 
 
 @pytest.mark.forked
 def test_remote_block_exact_match(atol_forward=1e-5, atol_inference=1e-3):
     dht = hivemind.DHT(initial_peers=INITIAL_PEERS, client_mode=True, start=True)
-    config = transformers.AutoConfig.from_pretrained(MODEL_NAME)
+    config = src.DistributedBloomConfig.from_pretrained(MODEL_NAME)
```

Comment:
```suggestion
    config = DistributedBloomConfig.from_pretrained(MODEL_NAME)
```

nit: Let's use a relative import, like the rest of code in this file does.

This is useful because people may copy the tests as code examples.